infra/base-images/base-builder/Dockerfile

```diff
@@ -57,11 +57,9 @@ RUN export PYTHON_DEPS="\
     rm -r /tmp/Python-$PYTHON_VERSION.tar.xz /tmp/Python-$PYTHON_VERSION && \
     apt-get remove -y $PYTHON_DEPS  # https://github.com/google/oss-fuzz/issues/3888
 
-# Install latest atheris for python fuzzing.
-RUN pip3 install atheris
-
-# Install pyinstaller for python fuzzer packaging.
-RUN pip3 install pyinstaller==4.1
+# Install latest atheris for python fuzzing, pyinstaller for fuzzer packaging.
+RUN export CFLAGS="" CXXFLAGS="" && \
```

Comment:
Any reason this needs to be done with export? Could this cause issues with any later commands?

Reply:
Yes removed, using directly.